README.md

```diff
@@ -609,6 +609,48 @@ Add the following job in your `azure-pipelines.yaml` file
           artifactName: MegaLinterReport
 ```
 
+If you want or need to have a central pipeline with a build validation branch policy covering all repositories then you need to make some modifications so the pipeline uses the correct git repository.
```

Comment:
Plz can you add headings for both cases, like:

### Pipeline in the repository

### Shared pipeline

Reply:
added and done some re-working too

Reply:
also updated ToC that I previously missed 🤦‍♀️